app/services/configuration_repository_service.rb

```diff
@@ -9,22 +9,30 @@ class ConfigurationRepositoryService
     include FastlaneCI::Logging
     include FastlaneCI::GitHubHandler
 
+    # An octokit client authenticated with the clone user's API token
+    #
+    # @return [Octokit::Client]
+    attr_reader :clone_user_client
```

Comment:
`clone_user_client` maybe should be `onboarding_user_client`?

Reply:
Hmm, I tried to keep it consistent with the environment variable names. Would we want to change the environment variable name then as well?

Reply:
Oyyyy maybe not right now.